### ISSUE TITLE
Feature request: Group calls

### ISSUE THREAD

**Issue by antonsteenvoorden** [overtake/telegram#460]
It's awesome that calls are now supported by the desktop app. 

It would be even better if this was also enabled for groups, where people can talk to more than 1 at a time and leave/join when they like.

I couldn't find another place to make a feature request, sorry if this isn't the method of doing it.

**Comment by paoletto** [comment]
It would be awesome.. There is currently no easy way to do it. Everything else is a pain for one reason or another.
And beside, if this feature would be there, many of us would have a VERY GOOD reason to force business partners or fellow gamers to install telegram (that is free) to just join a tcon because it works just better than the rest (Skype for business? I so hope i can uninstall that crap)

**Comment by antonsteenvoorden** [comment]
Yes it could even be peer-to-peer if server costs are an issue. I don't know how the current calls are implemented though
